src/Tribe/Aggregator/Service.php

```diff
@@ -335,7 +336,7 @@ public function get_eventbrite_token() {
 		 */
 		$args = apply_filters( 'tribe_aggregator_eventbrite_token_callback_args', $args );
 
-		$response = $this->get( 'eventbrite/token', $args );
+		$response = $this->get( 'eventbrite/validate', $args );
```

Comment:
It looks like this should be the endpoint to check if eventbrite has a token for the site, but I only get this message:

[12-Apr-2018 21:09:18 UTC] stdClass Object
(
    [status] => error
    [message_code] => error:eventbrite-token-not-found
    [message] => You do not have an active connection to Eventbrite through your account and Event Aggregator.
)

Route:
https://github.com/moderntribe/event-aggregator-site/blob/d3b1449144f9b1b6c6d1e6d23b4e890d81e49178/wp-content/plugins/event-aggregator/src/Tribe/Route/Eventbrite.php#L16

Reply:
Pass the license key as `key` and `url` (you have url param set correctly).

If you plan on supporting the TEC Eventbrite license key for this endpoint, you'll want to also pass in the `'origin' => 'eventbrite'`

Reply:
Sent you a DM in slack with the URLs that are working.